### PR TITLE
Add Skylark tab and show serial number and UUID in main window.

### PR DESCRIPTION
**Pull request by gsmcmullin**
For https://github.com/swift-nav/piksi_v3_bug_tracking/issues/1017

screenshot from 2018-03-16 09-59-59

The link is a button as that seems to be the only way to do it in Traits.
The UUID is shown in an editable text box so it can be selected, but if you try to edit it, it just changes back.

### CODE REVIEW

**Comment by mfine** [comment]
Sweet :+1:

**Comment by granthausler** [comment]
Thanks Gareth. Can we change the second sentence to:

"It removes the need for a base station or CORS station"  

(the reasoning here is that we don't want to reveal VRS as part of our service offering and the term is trademarked by trimble)

This all looks great. One other question is how difficult would it be to make another button saying 'Copy UUID' which copies the ID to clipboard when pressed. This is not a hard ask but just a thought.

File: piksi_tools/console/console.py [hunk 1]
```diff
@@ -357,6 +361,16 @@ class SwiftConsole(HasTraits):
...
                         style='readonly'),
+                    Item(
+                        '',
+                        label='Device UUID:',
+                        emphasized=True,
+                        tooltip='Corrections latency (-1 means no corrections)'
+                    ), Item(
+                        'uuid',
+                        padding=2,
+                        show_label=False,
```
Comment (denniszollo) [inline piksi_tools/console/console.py]: If you add editable=False here; then you won't be able to edit

**Comment by denniszollo** [comment]
We might be able to copy to the clipboard through pyside: https://srinikom.github.io/pyside-docs/PySide/QtGui/QClipboard.html

File: piksi_tools/console/skylark_view.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                    HGroup(spring, Item('link', show_label=False), spring),
+                    Item('uuid', label='Device UUID', width=400),
+                ), spring), spring))
+
+    def __init__(self):
+        self.real_uuid = ''
+
+    def _link_fired(self):
+        webbrowser.open(SKYLARK_URL)
+
```
Comment (denniszollo) [inline piksi_tools/console/skylark_view.py]: you won't need any of this below if you set editable=False

**Comment by denniszollo** [comment]
I'm also not thrilled to make another tab because it increased the minimum width of the widget and no longer lets you have a 600 Pixel console

**Pull request merged**